src/main/java/com/iota/iri/conf/PearlDiverConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.iota.iri.conf;
+
+/**
+ * Configurations for PearlDiver proof-of-work hasher.
+ */
+public interface PearlDiverConfig extends Config {
+
+    /**
+     * @return Descriptions#POW_THREADS
```

Comment:
```suggestion
     * @return {@value PowConfig.Descriptions#POW_THREADS}
```

There are currently documentation bugs in the new code :sad:
Not your fault that you made the mistake.
We need the fully qualified name for the IDE to generate the javadoc from whenever it is called.